### PR TITLE
Added Cloud Seeder

### PR DESCRIPTION
**Pull request by ipv6rs**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by kokomo123 — APPROVED** [review]
LGTM, pending further thoughts from collaborators, thank you for your contribution!

**Comment by ipv6rs** [comment]
Hello,

Hope you are all well and thanks again. I checked this and it seems like it's something on the build side versus the commit, but I'll defer to you.

Please let me know if there is something that I need to address and thank you in advance!

**Comment by kokomo123** [comment]
``ERROR:awesome_lint.py: Cloud Seeder: platforms Podman is not listed in the main platforms list``
Let us adjust this one.

File: software/cloud-seeder.xml
Comment (kokomo123) [inline software/cloud-seeder.xml:8]:
```suggestion
```
Does this software work with Docker? if so, you can add docker here. Otherwise, podman isn't added yet. that's why it errors out.

**Comment by ipv6rs** [comment]
Ah it currently runs with podman and not docker since there's certain specifics podman offers that docker does not. If it's easier we can remove the reference to podman or docker as the software will in fact prompt to get podman anyway!

Thank you again!

**Comment by kokomo123** [comment]
Just for this, I am going to create a PR that adds podman to the list so it's compatible from now on.

**Comment by ipv6rs** [comment]
Thank you so much!

File: software/cloud-seeder.xml
Comment (nodiscc) [reply software/cloud-seeder.xml:8]: Hi, as discussed in https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/856, I think it's better/easier just to remove `Podman` for the time being.

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here then!

**Review by nodiscc — APPROVED** [review]

**Pull request merged**
